classes/music_album.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'item'
+require 'date'
+
+class MusicAlbum < Item
+  attr_reader :id, :on_spotify
+
+  def initialize(publish_date, on_spotify)
+    super(publish_date)
```

Comment:
The constructor of the Item class also expects an **archived** parameter. Currently, we get the following error when we create a new **MusicAlbum**. Hence, can you please receive the **archived** parameter in the constructor of **MusicAlbum** and pass it to the constructor of **Item** 

![image](https://github.com/henokkhm/microverse-module-4-ruby-group-capstone/assets/110997325/a309ff32-fa43-4b5c-ae69-64c8bb746bb0)

Reply:
so i add it like this ?

def initialize(publish_date, on_spotify, archived)
    super(publish_date, archived)
    @on_spotify = on_spotify

Reply:
Yes, that is how you should do it.